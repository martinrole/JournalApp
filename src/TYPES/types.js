export const types = {
    login: '[Auth] Login',
    logout: '[Auth] Logout',

    uiLoadingOn: '[UI] Loading ON',
    uiLoadingOff: '[UI] Loading OFF',

    uiSetError: '[UI] Set Error',
    uiRemoveError: '[UI] Remove Error',

    notesAdd: '[Notes] New Note',
    notesActive: '[Notes] Set Active',
    notesLoad: '[Notes] Load Notes',
    notesUpdate: '[Notes] Update Note',
    notesFileUrl: '[Notes] Update Image URL',
    notesDelete: '[Notes] Delete Note',
    notesClear: '[Notes] Clear Notes'
}