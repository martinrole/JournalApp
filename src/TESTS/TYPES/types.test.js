import { types } from "../../TYPES/types"

const tipos = {
    login: '[Auth] Login',
    logout: '[Auth] Logout',

    uiLoadingOn: '[UI] Loading ON',
    uiLoadingOff: '[UI] Loading OFF',

    uiSetError: '[UI] Set Error',
    uiRemoveError: '[UI] Remove Error',

    notesAdd: '[Notes] New Note',
    notesActive: '[Notes] Set Active',
    notesLoad: '[Notes] Load Notes',
    notesUpdate: '[Notes] Update Note',
    notesFileUrl: '[Notes] Update Image URL',
    notesDelete: '[Notes] Delete Note',
    notesClear: '[Notes] Clear Notes'
}

describe('Pruebas archivo Types', () => {

    test('Debe de tener los types completos', () => {

        expect( types ).toMatchSnapshot()
        expect( types ).toEqual( tipos )
    })

})