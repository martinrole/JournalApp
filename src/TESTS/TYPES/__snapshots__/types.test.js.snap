// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas archivo Types Debe de tener los types completos 1`] = `
Object {
  "login": "[Auth] Login",
  "logout": "[Auth] Logout",
  "notesActive": "[Notes] Set Active",
  "notesAdd": "[Notes] New Note",
  "notesClear": "[Notes] Clear Notes",
  "notesDelete": "[Notes] Delete Note",
  "notesFileUrl": "[Notes] Update Image URL",
  "notesLoad": "[Notes] Load Notes",
  "notesUpdate": "[Notes] Update Note",
  "uiLoadingOff": "[UI] Loading OFF",
  "uiLoadingOn": "[UI] Loading ON",
  "uiRemoveError": "[UI] Remove Error",
  "uiSetError": "[UI] Set Error",
}
`;
